docs/specifications/tests/DNSSEC-TP/dnssec05.md

```diff
@@ -19,10 +19,12 @@ The domain name to be tested.
 
 ### Ordered description of steps to be taken to execute the test case
 
-1. Retrieve the DNSKEY RR set from the child zone.
-2. The DNSKEY algorithm numbers are derived from all the DNSKEY RRs.
-3. The algorithm numbers are matched with the IANA DNSKEY algorithm table.
-4. If any the algorithm numbers is unassigned, reserved, a private algorithm
+1. Obtain a set of name server IP addresses using [Method4] and [Method5].
+2. Create a DNSKEY query for the zone with DO flag set.
+3. Send the DNSKEY query over UDP to one the name server IP addresses.
```

Comment:
3b. If the name server does not reply or does not reply with the AA bit set, pick the next
IP from the list in 1 and rerun 3 with that IP. If there are no more IPs to use, this test case
fails.